superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx

```diff
@@ -223,6 +225,7 @@ const FilterBar: React.FC<FiltersBarProps> = ({
   }, [dashboardId, dataMaskAppliedText, history, updateKey, tabId]);
 
   const handleApply = useCallback(() => {
+    dispatch(logEvent(LOG_ACTIONS_CHANGE_DASHBOARD_FILTER, {}));
```

Comment:
Maybe `LOG_ACTIONS_CHANGE_DASHBOARD_FILTERS` (plural)?

Reply:
@michael-s-molina I though there was merit in keeping the action name the same for logging consistency.

Note that previously a single filter-box chart could actually have multiple filters so the singular term was also likely misleading then as well.